client/src/main/java/io/pravega/client/connection/impl/AppendBatchSizeTrackerImpl.java

```diff
@@ -27,19 +27,24 @@
  * synchronous writers. Otherwise the batch size is set to the amount of data that will be written in the next
  * {@link #MAX_BATCH_TIME_MILLIS} or half the server round trip time (whichever is less)
  */
-class AppendBatchSizeTrackerImpl implements AppendBatchSizeTracker {
-    private static final int MAX_BATCH_TIME_MILLIS = 100;
-
+public class AppendBatchSizeTrackerImpl implements AppendBatchSizeTracker {
```

Comment:
Please, revert this change as it may be inducing artifacts on the client latency and makes it difficult to compare with the `master` version. It would be great to have the batching-related changes on a separate PR.

Reply:
I don't think the changes here are function in nature. Please feel free to verify this.